src/commands/dora/__tests__/deployment.test.ts

```diff
@@ -0,0 +1,161 @@
+// import {default as axios} from 'axios'
+import {Cli} from 'clipanion/lib/advanced'
+
+import {createMockContext} from '../../../helpers/__tests__/fixtures'
+import * as gitFunctions from '../../../helpers/git/get-git-data'
+
+import {SendDeploymentEvent} from '../deployment'
+import {DeploymentEvent} from '../interfaces'
+
+describe('deployment', () => {
+  describe('getApiHelper', () => {
+    test('should throw an error if API key is undefined', () => {
+      process.env = {}
+      const write = jest.fn()
+      const command = new SendDeploymentEvent()
+      command.context = {stdout: {write}} as any
+
+      expect(command['getApiHelper'].bind(command)).toThrow('API key is missing')
+      expect(write.mock.calls[0][0]).toContain('DD_API_KEY')
+    })
+  })
+})
+
+const makeCli = () => {
+  const cli = new Cli()
+  cli.register(SendDeploymentEvent)
+
+  return cli
+}
+
+describe('execute', () => {
+  const runCLI = async (extraArgs: string[], extraEnv?: Record<string, string>) => {
+    const cli = makeCli()
+    const context = createMockContext() as any
+    process.env = {DD_API_KEY: 'PLACEHOLDER', ...extraEnv}
+    context.env = process.env
+    const code = await cli.run(['dora', 'deployment', ...extraArgs], context)
+
+    return {context, code}
+  }
+  describe('dry-run', () => {
+    const fakeCurrentDate = new Date(1699960651)
+    beforeAll(() => {
+      jest.useFakeTimers()
+      jest.setSystemTime(fakeCurrentDate)
+    })
+    afterAll(() => {
+      jest.useRealTimers()
+    })
+    test('with all parameters provided', async () => {
+      /* eslint-disable prettier/prettier */
+      const {context, code} = await runCLI([
+        '--dry-run',
+        '--service', 'test-service',
+        '--started-at', '1699960648',
```

Comment:
should we extract `1699960648` into its own variable and use 
```
TIME_VAR * 1000
```
and 
```
`${TIME_VAR}` 
```

Reply:
I gave it a shot but I'm not good enough with times in javascript to make this more readable without losing confidence in the tests 😂